### ISSUE TITLE
Manjaro unstable gnome 45

### ISSUE THREAD

**Issue by washran** [toppk/gnome-shell-extension-openweather#1]
This extension is causing brief stalling/freezing 10-30 seconds on starting up and after logout and login. Turning extension off there's no stalling or freezing.  Not sure you want issues being posted yet?

**Comment by avgsmoe** [comment]
Same issue on Ubuntu 23.10

**Comment by QuentiumYT** [comment]
Yes, looks like some memory leak happens, the gnome-shell RAM usage goes crazy until it has filled the swap which causes freezes
